types/Checkbox.php

```diff
@@ -62,8 +67,11 @@ public function multiValueEditor($name, $rawvalues) {
                 $checked = '';
             }
 
+            $id = !empty($htmlID) ? "id=\"$htmlID\"" : '';
+            $htmlID = null;
```

Comment:
Why the assignment to `$htmlID` here? It's not passed by reference and not used anymore further down the function?